### PR TITLE
Add Dependabot configuration

### PR DESCRIPTION
**Pull request by mikhail-shkaralevich**
# Ticket Link

[Ticket](https://uhawaii.atlassian.net/browse/GROUPINGS-1731)

# List of squashed commits

- Add Dependabot configuration

# Test Checklist

- [x] Exhibits Clear Code Structure:
- [x] Project Unit Tests Passed:
- [x] Project Integration Tests Passed:
- [x] Executes Expected Functionality:
- [x] Tested For Incorrect/Unexpected Inputs:

### CODE REVIEW

**Comment by JorWo** [comment]
I am curious if you have tested this to work on multiple branches.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+version: 2
+updates:
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "sunday"
+      time: "20:00"
+      timezone: "Pacific/Honolulu"
+    open-pull-requests-limit: 10
```
Comment (JorWo) [inline .github/dependabot.yml:10]: Since multiple branches are being defined, could you add `target-branch: "main"` to make it more clear? Thanks!

**Comment by mikhail-shkaralevich** [comment]
> I am curious if you have tested this to work on multiple branches. If not, I am fine with merging this as a test run.

Yes, I tested it. It does work for both branches.

File: .github/dependabot.yml (hunk 1 above)
Comment (mikhail-shkaralevich) [reply .github/dependabot.yml:10]: Before I do that, could you also read this: [link](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#target-branch). Do you understand what they mean by "When you use this option, the settings for this package manager will no longer affect any pull requests raised for security updates."? Doesn't seem obvious to me.

**Review by JorWo — APPROVED** [review]
Thanks for making sure Dependabot will work.

**Pull request merged**